### ISSUE TITLE
Feature Request: Supporting loading multiple Citation Database Paths

### ISSUE THREAD

**Issue by rastna12** [hans/obsidian-citation-plugin#245]
**Describe the solution you'd like**
My Better BibTex for Zotero plugin is configured to export multiple JSON files for my various libraries (personal, shared, etc.). It would be nice to be able to select and load multiple citation database paths into obsidian through this plugin.

**Describe alternatives you've considered**
I have a simple Python script that loads and combines these multiple JSON files into one file which the obsidian-citation-plugin then reads in. It would be nice if this step was part of the plugin itself.

**Comment by fweidner** [comment]
Second that, would be great!

**Comment by majdal** [comment]
+1!

**Comment by LiGurtner** [comment]
Yes, please, this would be great, I work with many shared libraries that are otherwise cumbersome to integrate...

**Comment by a2975667** [comment]
Adding my support. Perhaps instead of loading the first file, we can push all contents in the list of bib files into `this.library`?
I also think #200 is requesting the same feature.

**Comment by kiowafa** [comment]
+1 would also love this!

**Comment by atisu** [comment]
+1

**Comment by ankitp94** [comment]
+1

**Comment by finedust** [comment]
Would not be possible to pull directly all the Zotero's database as [Pandoc Reference List](https://github.com/mgmeyers/obsidian-pandoc-reference-list) does? I'm not sure if it uses the localhost API, Better BibTex or both though.
